<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-image: url(:/imagenes/FondoLogIn.png);
    background-repeat: no-repeat;
    background-position: center;
    background-color: rgba(0, 0, 0, 0.5);
}

QGroupBox::title {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Bienvenido a  MERPIDO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>97</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Correo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>92</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_userName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 10px;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_userPassword">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 10px;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logIn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px;
    padding: 10px 20px;
    background-color: #333333;
    color: #FFFFFF;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px;
    padding: 10px 20px;
    background-color: #333333;
    color: #FFFFFF;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperaturaActual">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: black; 
font-size: 16px; 
padding: 8px;
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string>Temperatura: </string>
    </property>
   </widget>
   <widget class="QLabel" name="ImagenTiempo">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Recordarme">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    color: black;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
    background-color: white /* Cambia #FF0000 al código de color deseado */;
   border-radius: 10px;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Recordarme</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
